en/flight_controller/durandal.md

```diff
@@ -144,6 +144,18 @@ To [build PX4](https://dev.px4.io/master/en/setup/building_px4.html) for this ta
 make holybro_durandal-v1_default
 ```
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+USART1 | /dev/ttyS0 | GPS1
+USART2 | /dev/ttyS1 | TELEM1
+USART3 | /dev/ttyS2 | TELEM1
+UART4 | /dev/ttyS3 | TELEM4
+USART6 | /dev/ttyS4 | ?
```

Comment:
Can't find what Duranda USART6 is used for.

Reply:
S0 CONFIG_STM32H7_USART1=y GPS
S1 CONFIG_STM32H7_USART2=y T1
S2 CONFIG_STM32H7_USART3=y T2
S3 CONFIG_STM32H7_UART4=y   T4 (GPS2)
S4 CONFIG_STM32H7_USART6=y T3
S5 CONFIG_STM32H7_UART7=y Debug 
S6 CONFIG_STM32H7_UART8=y  PX4IO